crates/kopuz/assets/themes.css

```
@@ -497,6 +497,207 @@ select:disabled {
     width: min(14rem, calc(100vw - 2rem));
 }
 
+.settings-workspace {
+    --settings-fan-radius: clamp(5rem, calc(50vh - 3.75rem), 15.5rem);
+    --settings-fan-radius: clamp(5rem, calc(50dvh - 3.75rem), 15.5rem);
```

Stylelint (17.14.1)

[error] Duplicate custom property "--settings-fan-radius" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)